File: web/src/model/victron_controller/dashboard/from_0_1_0_ws-ping.ts
// @ts-nocheck
import {WsPing as v0_1_0_WsPing} from './v0_1_0/WsPing'
import {WsPing as dashboard_WsPing} from './WsPing'

export function convert__ws_ping__from__0_1_0(from: v0_1_0_WsPing): dashboard_WsPing {
    return new dashboard_WsPing (
        from.nonce,
        from.client_ts_ms
    )
}

File: crates/dashboard-model/src/victron_controller/dashboard/from_0_1_0_ws_ping.rs


pub fn convert__ws_ping__from__0_1_0(from: &crate::victron_controller::dashboard::v0_1_0::ws_ping::WsPing) -> crate::victron_controller::dashboard::ws_ping::WsPing {
    crate::victron_controller::dashboard::ws_ping::WsPing {
        nonce: from.nonce.clone(),
        client_ts_ms: from.client_ts_ms.clone(),
    }
}